### ISSUE TITLE
autoscaleGroup not working in loadBedGraphTrack

### ISSUE THREAD

**Issue by paul-shannon** [gladkia/igvShiny#30]
@ChristianRohde  thank you for the bug report.   This feature works find in base igv.js, so the problem is mine to fix, in my code.

I put some time in on this just now, to no avail.  I need to dig deeper into the problem. Unfortunately I am tied up all week, so cannot get to this right away. 

I hope that the workaround I suggested [here](https://github.com/paul-shannon/igvShiny/issues/29#issuecomment-1141480799) is enough for now.

**Comment by paul-shannon** [comment]
@ChristianRohde 
I have not forgotten about this, long though it has been delayed.   I seem to have encountered and igv.js bug, or my own confusion in how Promises work.   This is reported on the igv.js [Issues](https://github.com/igvteam/igv.js/issues/1528).  I hope to hear back from the igv.js team soon, and to get this feature working again.

**Comment by paul-shannon** [comment]
Just heard back from Jim Robinson, https://github.com/igvteam/igv.js/issues/1528
```
Edit: the above will work to load the track, but group autoscale still doesn't 
work until something else forces an update. Group autoscale was apparently
 never tested for dynamic track loading (i.e. by calling loadTrack). So at this 
point it won't work, you will need to wait until the next release, probably by the 
end of the month or first week of July.
```

**Comment by gladkia** [comment]
Hi @ChristianRohde,

Can you please check if the issue still occurs? Thanks!

**Comment by ChristianRohde** [comment]
Hi @gladkia,

it seems that I ignored this issue which even might have started with one of my issue reports. I remember that these days I switched back my productive Apps to an older Version of igvShiny and kept things running. In the meantime things probably evolved. However, as I only install fresh if I have too like with new Hardware, a fresh R version or a new feature announced here, I got a bit lost in version numbers and updates. Right now I have trouble to understand the evolution of the issue: based on session_info() it looks like that I use igvShiny version 0.99.0 on my productive Servers with R 4.3.1 & 4.3.2. I think I installed the R versions somehow late 2023. Could this be true? I have trouble to understand the version number since we were talking about igvShiny 1.5.1 in 2022. Am I still stuck on a old version? At least with this version I have no trouble at the moment. 

Best,
Christian
